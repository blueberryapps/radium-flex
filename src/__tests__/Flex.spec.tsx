import { render } from 'enzyme';
importÂ  { StyleRoot } from 'radium';
import * as React from 'react';
import { Box, Flex } from '../index';

it('should render Flex with default style', () => {
  expect(render(<StyleRoot><Flex><Box>content</Box></Flex></StyleRoot>)).toMatchSnapshot();
});

it('should render Flex with falsey elements', () => {
  expect(render(<StyleRoot><Flex><Box>content</Box>{null}{false}</Flex></StyleRoot>)).toMatchSnapshot();
});
