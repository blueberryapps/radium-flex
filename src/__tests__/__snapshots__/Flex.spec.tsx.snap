// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Flex with default style 1`] = `
<div
  data-radium="true"
>
  <div
    data-radium="true"
    style="display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-start;align-items:stretch;align-content:stretch"
  >
    <div
      class="rmq-ce8f0f0d rmq-d31490d2 rmq-ed672573 rmq-5385cdb1 rmq-c847798b rmq-fe61d09d"
      data-radium="true"
      style="display:block;align-self:auto;flex-grow:0;flex-shrink:1;box-sizing:border-box;width:100%"
    >
      content
    </div>
  </div>
</div>
`;

exports[`should render Flex with falsey elements 1`] = `
<div
  data-radium="true"
>
  <div
    data-radium="true"
    style="display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-start;align-items:stretch;align-content:stretch"
  >
    <div
      class="rmq-ce8f0f0d rmq-d31490d2 rmq-ed672573 rmq-5385cdb1 rmq-c847798b rmq-fe61d09d"
      data-radium="true"
      style="display:block;align-self:auto;flex-grow:0;flex-shrink:1;box-sizing:border-box;width:100%"
    >
      content
    </div>
  </div>
</div>
`;
